package.json

```diff
@@ -113,6 +124,7 @@
     "mocha": "^8.2.1",
     "node-notifier": "^9.0.0",
     "normalize.css": "^8.0.1",
+    "npm-force-resolutions": "0.0.3",
```

Comment:
this was pulled a bit quicker than I was expecting - but we may not need these forced resolutions.  I'll run a check and see if we can take them out of the package.

Reply:
We can merge once this is verified.

Reply:
This was originally placed in to get around an error that is no longer present when we rebuild with the CLI.  We should remove
```suggestion
```
which would also include removing
```json
  "resolutions": {
    "chokidar": "^3.5.1",
    "webpack": "^5.20.2"
  }
```
and the preinstall
```json
  "preinstall": "npx npm-force-resolutions",
```
once those are gone, we would want to recompile and include the update `package-lock.json` 👍 

Thanks @mejiaj